### ISSUE TITLE
Can't build pyme-depends-strict for windows

### ISSUE THREAD

**Issue by David-Baddeley** [python-microscopy/pyme-conda-recipes#18]
```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {'readline==8.0', 'menuinst==1.14.6'}
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed
```

showing up as success though - so also need to look at the build script as well - there's something iffy there too.

**Comment by David-Baddeley** [comment]
see also https://github.com/python-microscopy/pyme-conda-recipes/runs/1969082722?check_suite_focus=true

**Comment by zacsimile** [comment]
I believe this is fixed in #19.
